src/elements/LabeledRange/LabeledRange.tsx

```diff
@@ -20,11 +22,13 @@ export class LabeledRange extends React.Component<
 > {
   static defaultProps = {
     disabled: false,
+    unit: "",
   }
 
   state = {
     min: this.props.defaultValue[0],
     max: this.props.defaultValue[1],
+    unit: this.props.unit,
```

Comment:
The properties for a component are defined when the component is created. Sometimes the properties pass down into the initial state for the component. A component's state tracks attributes that change over time.

For the `unit` properly, I don't think it's likely to change so shouldn't need to track it as state. You can refer to it within the component via `this.props.unit`.

Reply:
Makes sense! Still wrapping my head around props + state :)